### PR TITLE
entproto: add Skip annotation

### PR DESCRIPTION
**Pull request by iivankin**
Allows us to skip the generation of the specified fields and edges.

### CODE REVIEW

**Comment by rotemtam** [comment]
Hey,
Thanks for working on this.
Great start!
Can you please add a unit test in https://github.com/ent/contrib/blob/master/entproto/internal/entprototest/adapter_test.go
That asserts that skipped fields don't appear in the proto descriptor?

Thanks 🙏

**Comment by rotemtam** [comment]
Also CI is red , ping me on slack if you need assistance with getting it to pass

File: entproto/adapter.go [hunk 1]
```diff
@@ -321,6 +321,11 @@ func (a *Adapter) toProtoMessageDescriptor(genType *gen.Type) (*descriptorpb.Des
 	all = append(all, genType.Fields...)
 
 	for _, f := range all {
+		_, ok := f.Annotations[SkipAnnotation]
+		if ok {
+			continue
+		}
+
```
Comment (a8m) [inline entproto/adapter.go:328]:
```suggestion
		if _, ok := f.Annotations[SkipAnnotation]; ok {
			continue
		}
```

File: entproto/adapter.go [hunk 2]
```diff
@@ -337,6 +342,11 @@ func (a *Adapter) toProtoMessageDescriptor(genType *gen.Type) (*descriptorpb.Des
 	}
 
 	for _, e := range genType.Edges {
+		_, ok := e.Annotations[SkipAnnotation]
+		if ok {
+			continue
+		}
+
```
Comment (a8m) [inline entproto/adapter.go:349]:
```suggestion
		if _, ok := e.Annotations[SkipAnnotation]; ok {
			continue
		}
```

**Review by a8m — APPROVED** [review]
Looks good. minor style comments

**Pull request merged**